Lukman/employee

```diff
@@ -0,0 +1,21 @@
+const createEmployee = (name, role, salary) => {
+    return {name, role, salary};
+};
+
+let Lukman = createEmployee("Lukman", "Backend Dev", 5000);
+
+const getSalary = (employee) => {
+    return `The salary of ${employee.name} is ${employee.salary} per day`;
+};
+
+console.log(getSalary(Lukman));
+
+//update salary functioon
+let newSalary = (7000);
```

Comment:
Nice work lukman. But i will like to ask, why did you put the variable "7000" in a circle bracket? NOte you already got the assignment I am jus curios as to why you wrote that part like that.